Lecture 3/Task4/App.java

```diff
@@ -0,0 +1,64 @@
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.*;
+
+public class App {
+    private static List<Integer>arrayList;
+    private static List<Integer>linkedList;
+
+    public static void main(String[] args){
+
+        setup();
+
+        System.out.println("difference " + Math.abs(getTimeInsert(arrayList) - getTimeInsert(linkedList)) + "\n");
+
+        System.out.println("difference " + Math.abs(getTimeDelet(arrayList) - getTimeDelet(linkedList)) + "\n");
+
+        System.out.println("difference " + Math.abs(getTimeFind(arrayList) - getTimeFind(linkedList)) + "\n");
+
+
+
+    }
+    public static void setup(){
+        arrayList=new ArrayList<Integer>();
+        linkedList=new LinkedList<Integer>();
+
+        for (int i=0;i<1000000;i++ ) {
+            arrayList.add((int)(Math.random() * 10));
+        }
+        linkedList.addAll(arrayList);
+    }
+
+    private static long getTimeInsert(List<Integer> list) {
+        Date date0 = new Date();

```

Comment:
System.currentTimeMillis() is more efficient in this case as you don't need to create date object every time and you actually don't need Date type at all in this task